packages/edge-config/src/index.ts

```diff
@@ -114,10 +124,48 @@ export function createClient(
   if (typeof options.staleIfError === 'number' && options.staleIfError > 0)
     headers['cache-control'] = `stale-if-error=${options.staleIfError}`;
 
+  let websocketEdgeConfig: EmbeddedEdgeConfig | null = null;
+
+  // subscribe to changes
+  if (
+    process.env.NODE_ENV === 'development' &&
+    process.env.EDGE_CONFIG_DISABLE_WEBSOCKET !== '1' &&
+    typeof WebSocket !== 'undefined'
+  ) {
+    const ws = new WebSocket(
+      `wss://edge-config.vercel.com/websocket/connect?edgeConfigId=${connection.id}`,
+      {
+        // @ts-expect-error -- this is defined in undici which is used during dev
+        headers: { authorization: connection.token },
```

Comment:
It's not yet clear whether this will be the final way to authenticate, so this PR is still in draft mode until this is settled.